core/src/test/scala/unit/kafka/admin/ResetConsumerGroupOffsetTest.scala

```diff
@@ -87,6 +87,10 @@ class ResetConsumerGroupOffsetTest extends ConsumerGroupCommandTest {
     val args = Array("--bootstrap-server", brokerList, "--reset-offsets", "--group", group, "--all-topics",
       "--to-current", "--execute")
     val consumerGroupCommand = getConsumerGroupService(args)
+    // Make sure we got a coordinator
+    TestUtils.waitUntilTrue(() => {
+      consumerGroupCommand.collectGroupState().coordinator.host() == "localhost"
+    }, "Can't find a coordinator
```

Comment:
Was there a coordinator lookup issue that warranted this block? I couldn't get this test to fail without it.

Reply:
Do not have any idea really, this code appeared after merge from trunk

Reply:
Looks like the code has been committed by Gwen Shapira here to fix a test: 
KAFKA-7937: Fix Flaky Test ResetConsumerGroupOffsetTest.testResetOffsetsNotExistingGroup (#6311)

Reply:
@vahidhashemian is it ok if we leave it as is?

Reply:
@vahidhashemian one more thing: I'm unable to compile my branch anymore after merging with trunk due to a bunch of missing classes in imports section of **org.apache.kafka.common.protocol.ApiKeys.java** class like:
```
import org.apache.kafka.common.message.CreateTopicsRequestData;
import org.apache.kafka.common.message.CreateTopicsResponseData;
import org.apache.kafka.common.message.DescribeGroupsRequestData;
import org.apache.kafka.common.message.DescribeGroupsResponseData;
import org.apache.kafka.common.message.ElectPreferredLeadersRequestData;
import org.apache.kafka.common.message.ElectPreferredLeadersResponseData;
import org.apache.kafka.common.message.LeaveGroupRequestData;
import org.apache.kafka.common.message.LeaveGroupResponseData;
import org.apache.kafka.common.message.MetadataRequestData;
import org.apache.kafka.common.message.MetadataResponseData;
import org.apache.kafka.common.message.SaslAuthenticateRequestData;
import org.apache.kafka.common.message.SaslAuthenticateResponseData;
import org.apache.kafka.common.message.SaslHandshakeRequestData;
import org.apache.kafka.common.message.SaslHandshakeResponseData;
```
And I'm also unable to compile trunk as well for the same reason. 

Could we fix that?

Reply:
Regarding the added block of code I though you added it since it appeared in the commit diff. I now remember Gwen adding that block, so all is good. Sorry for the confusion.

I think I also noticed the compile issues you are referring to. Will have to dig further to determine the root cause.

Reply:
@vahidhashemian thanks for your accurate reviews, Vahid 👍 very straightforward and to the point